### ISSUE TITLE
Rename card and publish it to hacs

### ISSUE THREAD

**Issue by LesTR** [LesTR/homeassistant-minimalistic-area-card#128]
### What happened?

The original author does not communicate at all and the repo looks abandoned. [This PR](https://github.com/junalmeida/homeassistant-minimalistic-area-card/pull/105) is an example, my fork is currently 134 commits ahead with many improvements and fixes.

I want to rename the card (ideally with backward compatibility) and publish it as a new one to HACS. 

Suggested name: better-minimalistic-area-card.

I'll keep the credits to the original author in the readme, but I want to make it simpler for others to use.

Many issues/pull requests are closed as stale without any reaction.

### Card Version

not relevant

### HomeAssistant Version

not relevant

**Comment by LesTR** [comment]
Good, I was able to rename the card in https://github.com/LesTR/homeassistant-minimalistic-area-card/pull/137 and it looks like it was fully backward compatible. The users will see the following message in the javascript console:
```
better-minimalistic-…g=8313523241219:280 [DEPRECATED] You are using depracated card name 'custom:minimalistic-area-card', please update type to 'custom:better-minimalistic-area-card'. The old name will be removed in 1.3.0
```
and should change `type: 'custom:minimalistic-area-card'` to `type: 'custom:better-minimalistic-area-card'`.

The next step is publishing the card into hacs under the new name.

**Comment by LesTR** [comment]
@oerix, @RkcCorian, @LiquidPT, as far as I know, you are using my fork. Guys, can you please confirm if the transition to the new name was smooth?  

Just please update to v1.2.19, everything should still work as usual, but you should get the deprecated message in the js console (see above). That should disappear after changing the value of the type param.

**Comment by RkcCorian** [comment]
@LesTR : Hi! It is working but a HA restart was needed. It worked also backward compatible. Greetings!

**Comment by LesTR** [comment]
@RkcCorian; the restart is weird; probably just refreshing the cache should be enough. Maybe others can verify this.

Thanks for the confirmation; I'll open the PR into HACS tomorrow.

**Comment by kveldscholten** [comment]
I changed it with search and replace and it worked for me without restarting.

**Comment by LiquidPT** [comment]
@LesTR I took a quick look at your PRs into the HACS default repo, and I *think* the issue is that it should be in the `plugin` file

https://github.com/hacs/default/blob/master/plugin

If there were a `dashboard` file that would make sense, but I saw in [the HACS docs](https://hacs.xyz/docs/use/repositories/type/dashboard/) that dashboard elements used to be referred to as plugins.

What's more, the repo you forked from is in that plugin file.

**Comment by LesTR** [comment]
@LiquidPT, good catch! Thanks! It looks like you are right. Let's see. All those auto-closing bot actions are super annoying, hopefully now it will pass.

**Comment by simowilso** [comment]
Hi - Looks really nice, thank you,  is there any update on it being available in HACS,  and/or any hints how to move from the original minimalistic?

**Comment by LesTR** [comment]
@simowilso, sadly, no, the PR is still pending, and it will take some time as there are around 300 PRs in the queue before mine :-/ There is nothing that we can do, just wait.  But you can add my fork as a custom repo and use it as you are used to. If you don't know how you can check [this video](https://youtu.be/Y6YVQL0YHUw?t=57).

**Comment by LiquidPT** [comment]
Wow. The oldest items in the queue are a year old

**Comment by LesTR** [comment]
Cmon, it's just 4 clicks and all works exactly same after that. Updates, everything. Not so hard, just a new users are unable to find it 🤷‍♂️

**Comment by LesTR** [comment]
BTW originally, I opened PRs with all my changes to the source repo (just to make it clear), but all of them were closed by the stale bot without any comment. Even @junalmeida added the 👀 emoticon on some of my comments ([just one example](https://github.com/junalmeida/homeassistant-minimalistic-area-card/issues/110#issuecomment-2501907416)) , so he/she (not sure) must have seen it 🤷‍♂️

**Comment by simowilso** [comment]
> Cmon, it's just 4 clicks and all works exactly same after that. Updates, everything. Not so hard, just a new users are unable to find it 🤷‍♂️

Yeah you're right after I followed the vid it seems to be very straightforward, thanks

**Comment by LesTR** [comment]
Hehe, it's "just" 88 PR's in the queue before mine :D So maybe after two months

**Comment by LesTR** [comment]
<img width="506" alt="Image" src="https://github.com/user-attachments/assets/a7357857-2fa1-4895-8039-793750b39d01" />

Awesome, this card is now available via HACS! Closing and will update the readme later.
